website/views/options/inventory/seasonal-shop.jade

```diff
@@ -22,9 +22,7 @@
         p {{::Content.spells.special.seafoam.value}}
           span(class='shop_gold')
     menu.pets-menu(label='{{category.text}}', ng-repeat='category in seasonalShopCategories')
-      div(ng-repeat='item in category.items',
-        ng-class="{transparent: user.items.gear.owned[item.key] !== undefined}",
```

Comment:
I think this will make the item remain on the page. Keeping the `ng-if` should suffice to make it disappear.